testing/e2e/workload_test.go

```diff
@@ -316,3 +331,8 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 	}
 	testSuite.Run(t)
 }
+func foundMatch(pattern string, testCase string) bool {
+	regex := regexp.MustCompile(pattern)
```

Comment:
delete `foundMatch` func and call `FindString` inside the test case

Reply:
Took care in the latest commit